walkthroughs/howto-k8s-ingress-gateway/v1beta2/manifest.yaml.template

```diff
@@ -0,0 +1,621 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ${APP_NAMESPACE}
+  labels:
+    mesh: ${MESH_NAME}
+    gateway: ingress-gw
+    appmesh.k8s.aws/sidecarInjectorWebhook: enabled
+---
+apiVersion: appmesh.k8s.aws/v1beta2
+kind: Mesh
+metadata:
+  name: ${MESH_NAME}
+spec:
+  namespaceSelector:
+    matchLabels:
+      mesh: ${MESH_NAME}
+---
+apiVersion: appmesh.k8s.aws/v1beta2
+kind: VirtualGateway
+metadata:
+  name: ingress-gw
+  namespace: ${APP_NAMESPACE}
+spec:
+  namespaceSelector:
+    matchLabels:
+      gateway: ingress-gw
+  podSelector:
+    matchLabels:
+      app: ingress-gw
+  listeners:
+    - portMapping:
+        port: 8088
+        protocol: http
+      healthCheck:
+        path: /ping
+        healthyThreshold: 6
+        intervalMillis: 6000
+        protocol: http
+        timeoutMillis: 3000
+        unhealthyThreshold: 3
```

Comment:
We can remove this. Virtual Gateways currently do not have a routing health check endpoint, so it would result in health check failures anyway.

Current recommendation is to use a TCP health check from the NLB in front of it to determine routing health.

Reply:
👍